assets/l10n/app_en.arb

```diff
@@ -63,6 +63,18 @@
   "@actionSheetOptionUnstarMessage":  {
     "description": "Label for unstar button on action sheet."
   },
+  "errorAccountLoggedInTitle": "Account already logged in",
+  "@errorAccountLoggedInTitle": {
+    "description": "Error message when the account is already logged in."
```

Comment:
Same rewording as for the other description, but also "Error title" rather than "Error message".